src/AspNetCoreModuleV2/AspNetCore/applicationinfo.cpp

```diff
@@ -54,6 +39,7 @@ APPLICATION_INFO::GetOrCreateApplication(
             // Call to wait for application to complete stopping
             m_pApplication->Stop(/* fServerInitiated */ false);
             m_pApplication = nullptr;
+            m_pApplicationFactory = nullptr;
```

Comment:
Nit: helper method to call these 3 lines (repeated in shutdown application).

Reply:
![](https://media0.giphy.com/media/3o7btT1T9qpQZWhNlK/giphy.gif)

Reply:
k